app/javascript/components/App.js

```diff
@@ -316,6 +331,7 @@ export default class App extends React.Component {
             render={(props) => {
               let id = props.match.params.id
               let book = this.findItem(this.state.books, parseInt(id))
+              console.log("Lended:", book);
```

Comment:
🔥 Can be removed if not needed

Reply:
Nooo there's always at least one straggler haha. Thank you!